src/pretalx/orga/forms/review.py

```diff
@@ -0,0 +1,87 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from hierarkey.forms import HierarkeyForm
+from i18nfield.forms import I18nFormMixin
+
+from pretalx.common.forms import ReadOnlyFlag
+from pretalx.submission.models import Review
+
+
+class ReviewSettingsForm(I18nFormMixin, HierarkeyForm):
+    review_min_score = forms.IntegerField(
+        label=_('Minimum score'),
+        help_text=_('The minimum score reviewers can assign'),
+    )
+    review_max_score = forms.IntegerField(
+        label=_('Maximum score'),
+        help_text=_('The maximum score reviewers can assign'),
+    )
+
+    def __init__(self, obj, *args, **kwargs):
+        super().__init__(*args, obj=obj, **kwargs)
+        minimum = int(obj.settings.review_min_score)
```

Comment:
Opening the review settings gives me a

> TypeError at /orga/event/pw17/settings/reviews
> int() argument must be a string, a bytes-like object or a number, not 'NoneType'

maybe because there are no defaults defined for those settings?

Reply:
Yup, it is fixed!